modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path

```diff
@@ -160,11 +160,6 @@
 	<td>//*[@class='product-card-tiles']/div[${key_value}]</td>
 	<td></td>
 </tr>
-<tr>
-	<td>PRODUCT_CARD_DESCRIPTION</td>
-	<td>//p[@title='${key_productName}']/..//following-sibling::p[@class='card-body']//span[contains(normalize-space(text()),'${key_productDescription}')]</td>
-	<td></td>
-</tr>
```

Comment:
Why this path it was removed and readded in another location in the same .path file?

Thanks!